Tests/scs-compatible-iaas.yaml

```diff
@@ -6,7 +6,7 @@ versions:
     obsoleted_at: 2023-10-31
     standards:
       - name: Flavor naming
-        url: https://raw.githubusercontent.com/SovereignCloudStack/standards/main/Drafts/flavor-naming.md
+        url: https://raw.githubusercontent.com/SovereignCloudStack/standards/main/Standards/scs-0100-v1-flavor-naming.md
```

Comment:
why not use https://raw.githubusercontent.com/SovereignCloudStack/standards/1745d4b0f8f1b1a5558c8bb5d89f0e3124e42e0a/Standards/scs-0100-v1-flavor-naming.md